### PR TITLE
Scraper App. This app is implemented on core Ruby Language. It is the for extract of content from web pages.

### PR DESCRIPTION
**Pull request by promise-J**
This project aims to develop a Web Scraper using Ruby. It is a capstone project that marks the completion of Ruby module in Microverse technical curriculum.
I adhered strictly to
 - Creating effective Ruby code, that solves the problem
- Applying Ruby coding best practices.
- Creating unit test for Ruby code
- Implementing basic OOP principles (encapsulation, inheritance) in Ruby
- Having a organized project structure: modules, files and folders
- Using common Ruby patterns
- Choosing a simple solution over a more complex answer wherever possible.
- Multitasking and effectively managing time and prioritization.
- Translating business requirements into software solutions.
- Performing tasks and complete projects with minimal supervision.

### CODE REVIEW

**Review by DeliceLydia — COMMENTED** [review]
# Status: Invalid

Hello @promise-J  we can see you worked hard on this project but try to make
your repo looking professionally by adding

- descriptive README
- A meaningful title to your PR and a description
- And try to write a commit message anytime you added a meaningful change

After making all these changes submit for the review, and keep working hard!

**Review by taiwo2 — CHANGES_REQUESTED** [review]
Hello @promise-J we can see you worked hard on this project, kindly implement what the previous TSE suggested

After making all these changes submit for the review, and keep working hard!

**Comment by elisha2kyakpo1** [comment]
I can see you have worked really hard, work on the changes mentioned by @DeliceLydia and submit another request.

File: lib/scraper.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+    title_cat
+  end
+
+  def course1
+    full_page.css('.container-fluid div.row .col-md-4 .bg-black span.font-alt').text
+  end
+
+  def course2
+    full_page.css('section#youtube .container .col-md-6 .bg-white span').text.gsub("\n", '')
```
Comment (henatan99) [inline lib/scraper.rb:35]: This methods are returning a single string and it is very difficult to distinguish between the different titles. Please use enumerators to have an object with array of strings.

Comment (promise-J) [reply lib/scraper.rb:35]: I have solve the loop issues when a user choose 'NO'. I have also fixed the rspec testing units to act as supposed.

Comment (henatan99) [reply lib/scraper.rb:35]: - Hi @promise-J, I have run the cloned repo of your project. I have attached a screenshot to show you how the console output is displayed. 
Screenshot from 2021-03-26 01-59-26
- Very sorry for the spec comment. There was a bad network when I run Rspec locally and it gave me errors. But the first comment needs to be addressed.

**Pull request merged**